src/pre_defined_classes/programs_with_changing_behaviour/hard/q2/Question2Test.java

```diff
@@ -0,0 +1,25 @@
+package pre_defined_classes.programs_with_changing_behaviour.hard.q2;
+
+import global.BaseRandomTest;
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.RandomInteger;
+import global.variables.clauses.StringLiteral;
+
+public class Question2Test extends BaseRandomTest {
```

Comment:
I feel like we should just name this class `MainTest` always since this is what it needs to be called when we paste them into the website

Reply:
Should I name it MainTest instead of whatever the class is called in the question?

Reply:
`MainTest` is the move